pkgs/by-name/fl/flow-control/update.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -i bash -p bash common-updater-scripts gnused nixfmt-rfc-style
+#!nix-shell -i bash -p bash common-updater-scripts gnused nixfmt
```

Comment:
This shebang is using the impure `NIX_PATH` with no `-I nixpkgs=.`; so packages will come from whatever nixpkgs channel happens to be configured on the user's system.